core/arch/arm/arm.mk

```diff
@@ -18,3 +18,10 @@ core-platform-cppflags += $(arm32-platform-cppflags)
 core-platform-cflags += $(arm32-platform-cflags)
 core-platform-aflags += $(arm32-platform-aflags)
 endif
+
+ifeq ($(CFG_WITH_PAGER),y)
+ifneq ($(CFG_DISABLE_CONCURRENT_EXEC),y)
```

Comment:
@jforissier what about extending `force` with an optional string to print in case of error, something like:

    $(call force,CFG_DISABLE_CONCURRENT_EXEC,y,"required by CFG_WITH_PAGER")